### DISCUSSION TITLE
Execution failed for task ':app:minifyReleaseWithR8'.

### DISCUSSION THREAD

**Discussion by Roni27Patel** [pichillilorenzo/flutter_inappwebview#2338] [Q&A]
* What went wrong:
Execution failed for task ':app:minifyReleaseWithR8'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.R8Task$R8Runnable
   > Compilation failed to complete, origin: M:\Kevalam\Flutter-Projects\slyce-git-2023\build\flutter_keyboard_visibility\intermediates\runtime_library_classes_jar\release\classes.jar:com/slyce/gourmet/BuildConfig.class

**Comment by Enisco** [comment]
Found a way to solve this?
